### PR TITLE
BUGZ-736 / BUGZ-914 : Improve the enumeration of the platform and profiler / apply a specific PPI per Tier 

### PR DESCRIPTION
**Pull request by samcake**
https://highfidelity.atlassian.net/browse/BUGZ-736
https://highfidelity.atlassian.net/browse/BUGZ-914

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20252/HighFidelity-Beta-PR15884-0c26d3e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20252/HighFidelity-Beta-Interface-PR15884-0c26d3e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20252/HighFidelity-Beta-PR15884-0c26d3e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20252/HighFidelity-Beta-Interface-PR15884-0c26d3e.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20252/HighFidelity-Beta-PR15884-0c26d3e.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20252/HighFidelity-Quest-Beta-PR15884-0c26d3e.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20253/HighFidelity-Beta-PR15884-9d9b80d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20253/HighFidelity-Beta-Interface-PR15884-9d9b80d.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20253/HighFidelity-Beta-PR15884-9d9b80d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20253/HighFidelity-Quest-Beta-PR15884-9d9b80d.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20254/HighFidelity-Beta-PR15884-6ef6520.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20254/HighFidelity-Beta-Interface-PR15884-6ef6520.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20254/HighFidelity-Beta-PR15884-6ef6520.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20254/HighFidelity-Quest-Beta-PR15884-6ef6520.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20255/HighFidelity-Beta-PR15884-11f0362.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20255/HighFidelity-Beta-Interface-PR15884-11f0362.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20255/HighFidelity-Beta-PR15884-11f0362.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20255/HighFidelity-Quest-Beta-PR15884-11f0362.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20257/HighFidelity-Beta-PR15884-1aa067d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20257/HighFidelity-Beta-Interface-PR15884-1aa067d.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20257/HighFidelity-Beta-PR15884-1aa067d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20257/HighFidelity-Quest-Beta-PR15884-1aa067d.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20257/HighFidelity-Beta-PR15884-1aa067d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20257/HighFidelity-Beta-Interface-PR15884-1aa067d.exe

**Comment by JeffClinton** [comment]
Pre-merge testing PASS
- Clean install onto MBP High Sierra correctly defaults to mid/interactive/deferred/shadows off
- Resolution scale is 0.62 instead of 1.00
- High and Low tiers have expected settings
- You don't even have to refresh the script to see the changes!

Screen Shot 2019-07-05 at 11 25 39 AM

File: libraries/shared/src/GPUIdent.cpp [hunk 1]
```diff
@@ -289,7 +289,7 @@ GPUIdent* GPUIdent::ensureQuery(const QString& vendor, const QString& renderer)
 
                 nlohmann::json output = {};
                 output["model"] = deviceName;
-                output["coordinatesleft"] = test->DesktopCoordinates.left;
+                output["coordsinatesleft"] = test->DesktopCoordinates.left;
```
Comment (hyperlogic) [inline libraries/shared/src/GPUIdent.cpp]: wat? why did you change this? it should remain "coordinatesleft".

Comment (samcake) [reply libraries/shared/src/GPUIdent.cpp]: my bad

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20265/HighFidelity-Beta-PR15884-b4e8674.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20265/HighFidelity-Beta-Interface-PR15884-b4e8674.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20265/HighFidelity-Beta-PR15884-b4e8674.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20265/HighFidelity-Beta-Interface-PR15884-b4e8674.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20265/HighFidelity-Beta-PR15884-b4e8674.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20265/HighFidelity-Quest-Beta-PR15884-b4e8674.apk)
